src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -96,7 +96,7 @@ public static function collectDeprecations($outputFile)
     {
         $deprecations = [];
         $previousErrorHandler = set_error_handler(function ($type, $msg, $file, $line, $context = []) use (&$deprecations, &$previousErrorHandler) {
-            if (\E_USER_DEPRECATED !== $type && \E_DEPRECATED !== $type && (\E_WARNING !== $type || false === strpos($msg, '" targeting switch is equivalent to "break'))) {
+            if (\E_USER_DEPRECATED !== $type && \E_DEPRECATED !== $type && (\E_WARNING !== $type || !str_contains($msg, '" targeting switch is equivalent to "break'))) {
```

Comment:
I'm reverting all those, the phpunit-bridge doesn't support php and we don't want it to use polyfills

Reply:
@nicolas-grekas , i'm curious then, as bridge is already using 8.1 polyfill. Why not using 8.0 polyfill ?

Reply:
After checking, this is needed for testing the enum_exists mock. Also, this is require-dev!